src/sentry/tasks/embeddings_grouping/backfill_seer_grouping_records_for_project.py

```diff
@@ -223,50 +209,40 @@ def backfill_seer_grouping_records_for_project(
         },
     )
     call_next_backfill(
-        last_processed_group_index=batch_end_index,
+        last_processed_group_id=batch_end_id,
         project_id=current_project_id,
         redis_client=redis_client,
-        len_group_id_batch_unfiltered=total_groups_to_backfill_length,
-        last_group_id=last_group_id,
         last_processed_project_index=last_processed_project_index,
         cohort=cohort,
     )
 
 
 def call_next_backfill(
     *,
-    last_processed_group_index: int | None,
+    last_processed_group_id: int | None,
     project_id: int,
     redis_client: RedisCluster | StrictRedis,
-    len_group_id_batch_unfiltered: int,
-    last_group_id: int | None,
     last_processed_project_index: int,
     cohort: str | list[int] | None = None,
     only_delete: bool = False,
 ):
-
-    if last_group_id is not None:
+    if last_processed_group_id is not None:
         redis_client.set(
-            f"{make_backfill_grouping_index_redis_key(project_id)}",
-            last_processed_group_index if last_processed_group_index is not None else 0,
+            f"{make_backfill_grouping_id_redis_key(project_id)}",
```

Comment:
```suggestion
            make_backfill_grouping_id_redis_key(project_id),
```
it's already a string